spec/factories/users.rb

```diff
@@ -0,0 +1,5 @@
+FactoryBot.define do
+  factory :user do
+    password { 'password' }
```

Comment:
You probably need to add an email attribute to make the factory work (and add a spec for it - see comment below)

Reply:
done